docs/components/servo/pi.md

```diff
@@ -28,38 +28,9 @@ Click the **+** icon next to your machine part in the left-hand menu and select
 Select the `servo` type, then select the `pi` model.
 Enter a name for your servo and click **Create**.
 
-{{< imgproc src="/components/servo/pi-servo-ui-config.png" alt="An example configuration for a pi servo in the Viam app Config Builder." resize="600x" >}}
+![An example configuration for a pi servo in the Viam app Config Builder.](/components/servo/pi-servo-ui-config.png)
```

Comment:
ooh should these all be standardized? Is there a reason you changed this such that I should be using the MD syntax instead of imgproc for my component screenshot update PRs or is it arbitrary and doesn't matter?

Reply:
I changed it because I noticed an issue with resizing (image not shrinking with page) on the mmovement sensor pr images-- looked over it with Naomi and `resize="1200x" style="width:650px"` instead of `max-width` fixes the issue and allows the images to be slightly bigger-- would suggest that. I think either way is fine but imgproc resized looks a bit better than markdown so I'm going to change that back here